modifications/DataClass.cc

```diff
@@ -172,14 +233,246 @@ void DataClass::uploadPlanToServer(QString location, QJsonObject plan){
 void DataClass::readyReadFlightPlan()
 {
     QNetworkReply* reply = qobject_cast<QNetworkReply*>(sender());
-    QString replyStr = reply->readAll();
+    QString replyStr = reply->readAll();     
     int statusCode = reply->attribute( QNetworkRequest::HttpStatusCodeAttribute).toInt();
     if(statusCode == 201 || statusCode == 200){
+        QJsonDocument jsonReply = QJsonDocument::fromJson(replyStr.toUtf8());
+        QJsonObject jsonObjectReply = jsonReply.object();
+        showMessage("Flight Plan Uploaded Successfully", -1);
         emit planUploadSuccessful();
+        getAllFlightPlans();
     }else{
+        showMessage("Unable to Upload Flight Plan", statusCode);
         emit planUploadFailed();
     }
 }
+//*******************************************************************************************************************************************************************************
+// GET ALL FLIGHT PLANS
+void DataClass::getAllFlightPlans()
+{
+    QString location = serverUrl + AerobridgeGlobals::getAllFlightPlansUrl;
+    QNetworkRequest request = QNetworkRequest(location);
+    request.setRawHeader("Authorization",QByteArray("Bearer ").append(accessToken));
+    QNetworkReply* reply = manager.get(request);
+    connect(reply,&QNetworkReply::finished, this, &DataClass::readyReadAllFlightPlans);
```

Comment:
maybe its a Qt thing but to me its not clear how the get method doesn't return anything, what would the connect call do in this case? like you got all the flight plans and then? Display in UI? or then call readyReadAllFlightPlans? 

A one line top comment for these methods could help explain even if you don't rename them as such.